Specs/Sequence/Portuguese/IpAddressModel.json

```diff
@@ -1,6 +1,7 @@
 [
   {
     "Input": "O IP do meu computador é 1.1.1.1",
+    "NotSupported":"java",
```

Comment:
Is this because of the commented out regex? If so, then we should not register the IP recognizer yet. Please disable it on the previous PR until we have a solution.

Reply:
We updated the specs that are supported in Java Sequence. This PR contains the extractor of English and some specs are not supported because those regex are not configured (e.g. some specs of Chinese PhoneNumber model).

The current number of specs was improved to 845 passing correctly and 289 ignored.